### DISCUSSION TITLE
check logfile rotation

### DISCUSSION THREAD

**Discussion by CristianGLepore** [Linuxfabrik/monitoring-plugins#714] [Q&A] [answered]
Hello everyone. I am trying to use the logfile plugin to analyze the log files of my laravel installation.
The problem is that currently on my server we are using laravel files with a rotation of the type:

laravel-YYYY-MM-AA.log

What would be the best way to use this plugin contemplating this rotation?

Best regards.

**Comment by markuslf** [comment, marked as answer]
Cristian, does this open issue fit your topic? https://github.com/Linuxfabrik/monitoring-plugins/issues/678

**Reply by CristianGLepore** [reply]
Markus, that's exactly the problem I have. I will wait for the improvements.

Thanks a lot.

**Comment by markuslf** [comment]
Perfect, closing this discussion.
